os/arch/arm/src/armv7-a/arm_unblocktask.c

```diff
@@ -113,6 +113,9 @@ sched_removeblocked(tcb);
            */
 
           rtcb = this_task();
+
+	  /* Restore rtcb data for context switching */
```

Comment:
This kind of change should be in 1st commit, not 2nd commit.

Reply:
Updated, thankyou